tests/tests.rs

```diff
@@ -200,3 +200,47 @@ a,b,c
 "#
     );
 }
+
+#[test]
+fn test_null_byte_as_empty_cell() {
+    let testdir = TestDir::new("scrubcsv", "null_byte_as_empty");
+    let output = testdir
+        .cmd()
+        .arg("--null-byte-as-empty")
+        .output_with_stdin(
+            br#"c1,c2,c3
+1,2,3
+\x00,\x00,1
```

Comment:
this is failing because the `\x00` is being escaped to `\\x00`, and I'm not quite sure how to fix it. [the docs imply](https://doc.rust-lang.org/reference/tokens.html#raw-byte-string-literals) that using `br#` should work.